### PR TITLE
Add Only make the timer change button available when you are the owner of the group

### PR DESCRIPTION
**Pull request by Xiangrui2019**

### CODE REVIEW

**Comment by Anduin2017** [comment]
@xxyzz please review this.

**Review by hv0905 — CHANGES_REQUESTED** [review]
You should also check if the conversation is a `PrivateConversation`,

**Comment by Anduin2017** [comment]
You shall not close the issue #518. Because only change here does not satisfy its requirements.

File: src/app/Views/group.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 </div>
 
 <div class="menu-holder"></div>
-<div class="menu-button" *ngIf="conversation && conversation.ownerId == messageService.me.id" (click)="timerService.setTimer(conversation.id)">
+<div class="menu-button" *ngIf="conversation && conversation.discriminator == 'PrivateConversation' && conversation.ownerId == messageService.me.id" (click)="timerService.setTimer(conversation.id)">
```
Comment (hv0905) [inline src/app/Views/group.html]: use `||` instead of `&&`..........

**Review by hv0905 — CHANGES_REQUESTED** [review]

**Comment by Anduin2017** [comment]
BTW, I am not asked to delete this icon if you can't change it.

Still, display it. But don't make it clickable when you are not permitted to do so.

**Pull request closed without merging**